app/v2/hateoas/HateoasLinks.scala

```diff
@@ -125,7 +128,18 @@ trait HateoasLinks {
     Link(href = ukHistoricNonFhlAnnualUri(appConfig, nino, taxYear), method = DELETE, rel = DELETE_HISTORIC_NON_FHL_UK_PROPERTY_ANNUAL_SUBMISSION)
   }
 
-  // Generic
+
+  //Historic Uk PIE Period Summary
```

Comment:
If it's a comment, can it include the full name instead of PIE?
Just thinking of new devs looking at this in 2 years' time :)